Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM ros:melodic
+
+SHELL ["/bin/bash", "-c"]
+
+RUN useradd --create-home --shell /bin/bash spaceuser \
+    && mkdir /home/spaceuser/robotics-prototype
+
+WORKDIR /home/spaceuser/robotics-prototype
+COPY requirements.txt requirements-dev.txt setup.py ./
```

Comment:
I suggest to use /home/spaceuser/Programming/robotics-prototype
We have a file with handy dandy aliases like `robotics` that `cd`s to the robotics directory, or `cm` that `cd`s into the rospackages and runs catkin_make, and a lot more aliases, but they currently assume that you have a `Programming` folder like this `~/Programming/robotics-prototype`

Reply:
So this is done. See fff03f2